its/src/test/java/com/sonar/it/jenkins/RunOnlyOnWindows.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Jenkins :: Integration Tests
+ * Copyright (C) 2013-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.jenkins;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.internal.runners.InitializationError;
+import org.junit.internal.runners.JUnit4ClassRunner;
+import org.junit.runner.notification.RunNotifier;
+
+public class RunOnlyOnWindows extends JUnit4ClassRunner {
```

Comment:
This class `JUnit4ClassRunner` is marked as deprecated, and it says that we should use `BlockJUnit4ClassRunner`, have you tried that?